test/units/weight_test.rb

```diff
@@ -0,0 +1,80 @@
+require "test_helper"
+
+class Measured::WeightTest < ActiveSupport::TestCase
+  setup do
+    @weight = Measured::Weight.new(1, "g")
+  end
+
+  test ".units_with_aliases should be the expected list of valid units" do
+    assert_equal ["g", "gram", "grams", "kg", "kilogram", "kilograms", "lb", "lbs", "ounce", "ounces", "oz", "pound", "pounds"], Measured::Weight.units_with_aliases
+  end
+
+  test ".units should be the list of base units" do
+    assert_equal ["g", "kg", "lb", "oz"], Measured::Weight.units
+  end
+
+  test ".convert_to from g to g" do
+    assert_conversion Measured::Weight, "2000 g", "2000 g"
+  end
+
+  test ".convert_to from g to kg" do
+    assert_conversion Measured::Weight, "2000 g", "2 kg"
+  end
+
+  test ".convert_to from g to lb" do
+    assert_conversion Measured::Weight, "2000 g", "0.4409245244E1 lb"
+  end
+
+  test ".convert_to from g to oz" do
+    assert_conversion Measured::Weight, "2000 g", "70.547923904 oz"
```

Comment:
Quantity[2000.0000001, "Grams"] http://www.wolframalpha.com/input/?i=70.547923904+oz+in+g
Hmm. I wonder if it's because of the 1 oz = 0.45359237⁄16 kg rational

`In the international yard and pound agreement of 1959 the United States and countries of the Commonwealth of Nations agreed to define the international avoirdupois ounce to be exactly 0.45359237⁄16 kg (28.349523125 g) by definition.` http://en.wikipedia.org/wiki/Ounce